astropy/io/fits/tests/test_fitsheader.py

```diff
@@ -0,0 +1,122 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import sys
+
+import numpy as np
+import re
+
+from . import FitsTestCase
+from ..hdu import PrimaryHDU
```

Comment:
Please remove the imports that are not used (`PrimaryHDU`, `AstropyDeprecationWarning`).